### PR TITLE
prepare for release 20.04

### PR DESCRIPTION
**Pull request by clobrano**
Update changelog for release 20.04

### CODE REVIEW

**Comment by Feichtmeier** [comment]
Sorry, just merged a automation pr. Will keep my hands calm now so you don't need to change the changelog

**Comment by clobrano** [comment]
Ok

**Comment by clobrano** [comment]
Ready for review and merge, then I can create the tag. The only thing missing is the gpg key to sign it

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,212 @@
...
+  [ MadsRH ]
+  * Fix symlink badge for new folder icons (PR for 1742)
+  * Added cursor test
+  * Update README.md
+  * Mimetype icons for .PO translation files
+  * Mimetype icon for RSS
+  * Mimetype for application-x-python-bytecode to replace the Humanity fallback
+  * fixed missing "eyes and stroke covering the Python logo
+  * more contrast for context menus
+  * reverted popover
```
Comment (iainlane) [inline debian/changelog]: IMHO many of these entries don't make sense to someone reading the Debian changelog. Imagine if you have `apt-listchanges` installed and you are shown this when you do an upgrade. :)

I think it'd be best if there was something more descriptive - what changed that people will see? If there was a `NEWS` file announcing this new version, what would be in it? - or at least please could you delete the entries that aren't understandable like this one and many of the others?

Thanks!

Comment (clobrano) [reply debian/changelog]: Yeah, I understand what you mean. I'll try to rewrite some entries

**Comment by didrocks** [comment]
We discussed yesterday about me pushing the changes, but it seems Laney already jumped on it, feel free to handle the release then!

**Pull request merged**